### PR TITLE
Interact docs

### PR DESCRIPTION
**Pull request by LadyNaggaga**
Cleaned up the docs .

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -13,9 +13,7 @@ Please provide as much information as you can.
 - [ ] .NET Jupyter Notebook 
 
 #### What kind of error was it?
```
Comment (jonsequitur) [inline .github/ISSUE_TEMPLATE/bug_report.md]: I think we can do away with this heading now.

Comment (LadyNaggaga) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Agreed.

File: docs/CreateBinder.md [hunk 2]
```diff
@@ -62,4 +62,4 @@ into this:
 
 ```[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/dotnet/try/master?urlpath=lab)```
```
Comment (colombod) [inline docs/CreateBinder.md]: this needs to point to interactive, not to try

Comment (LadyNaggaga) [reply docs/CreateBinder.md]: Fixed

**Pull request merged**